app/views/layouts/_shim.html.erb

```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js">
+  </script>
+<![endif]-->
```

Comment:
please remove this file because not used yet

Reply:
not used yet?

Reply:
oke, no problems